example/lib/screens/icons.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:clipboard/clipboard.dart';
+import 'package:fluent_ui/fluent_ui.dart';
+
+class IconsPage extends StatefulWidget {
+  const IconsPage({Key? key}) : super(key: key);
+
+  @override
+  _IconsPageState createState() => _IconsPageState();
+}
+
+class _IconsPageState extends State<IconsPage> {
+  String filterText = "";
+
+  @override
+  Widget build(BuildContext context) {
+    assert(debugCheckHasFluentTheme(context));
+    return ScaffoldPage(
+      header: PageHeader(
+        title: const Text('Fluent Icons Gallery showcase'),
+        commandBar: SizedBox(
+          width: 240.0,
+          child: Tooltip(
+            message: 'Filter by name',
+            child: TextBox(
+              header: 'Search',
```

Comment:
I don't like this header + I think it's unecessary. What do you think?

Reply:
When I use FluentUI in my projects I always use headers on TextBoxes. 

As you wish, please feel free to remove it if you don't like it (you have write access in my fork when this PR is open)